nvflare/private/fed/app/client/worker_process.py

```diff
@@ -88,17 +89,17 @@ def main():
                 )
             )
 
-        workspace = os.path.join("/tmp/fl", client_name)
+        startup = args.startup
         app_root = os.path.join(args.workspace, "run_" + str(run_number), "app_" + client_name)
 
         app_log_config = os.path.join(app_root, config_folder, "log.config")
         if os.path.exists(app_log_config):
             args.log_config = app_log_config
         else:
-            args.log_config = os.path.join(workspace, "log.config")
+            args.log_config = os.path.join(startup, "log.config")
 
         conf = FLClientStarterConfiger(
-            app_root=workspace,
+            app_root=startup,
```

Comment:
Maybe the only arg needed is the "workspace". Since its structure is standard, all other files can be found there following the folder structure.